wordpress-mobile/WordPress-iOS
# Jetpack overlay - left alignment and copy update

Fixes #19155

This PR updates the layout of the Jetpack overlay to be aligned to the leading edge, and updates the copy as well (see animation below)

<p align=center>
<img height=440 width=390 src=https://user-images.githubusercontent.com/34376330/184047662-78cb908a-25bf-4d76-8134-734a37ab8055.gif>
</p>


To test:
Test with one LTR and one RTL language
- Enable the `jetpackBottomSheet` feature flag
- build/run this branch
- go to My Site/Home
- tap the Jetpack powered badge
- Make sure the overlay appears and its content is aligned to the leading edge (left for a LTR, right for a RTL)
- Make sure the copy looks correct.


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
